docs/resolving-link-and-linting-issues.md

```diff
@@ -0,0 +1,82 @@
+# Resolving Link and Linting Issues in Contributions
+
+To maintain the quality and reliability of our content, we've integrated linting
+and link validation checks into our GitHub Actions CI/CD pipeline. This document
+guides you through identifying issues flagged by these checks and resolving them
+using our `Makefile` commands.
+
+## Understanding the Checks
+
+Our CI/CD pipeline includes two primary checks for content contributions:
+
+1. **Markdown Linting**: Ensures that Markdown files adhere to our project's
+   formatting standards.
+2. **Link Validation**: Checks for broken links in the compiled site output to
+   ensure all internal and external links are valid.
+
+If either of these checks fails during a GitHub Actions run, your pull request
+will indicate a failure, requiring resolution before merging.
+
+## Resolving Issues
+
+### Markdown Linting Issues
+
+If the linting check fails, it typically means there are formatting
+discrepancies in your Markdown files.
+
+- **Identify Issues**: Review the output of the GitHub Actions run to see which
+  files and specific formatting rules were violated.
+- **Auto-Format Locally**: Run the following command in your local environment
+  to automatically format all Markdown files according to our project's
+  standards:
+
+  `bash make format-md`
```

Comment:
I'm just wondering how easy this is going to be for non-devs adding new resources via the GitHub UI. Does the linter fail a PR for line length, for example?